dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -428,6 +405,81 @@ QUERY
     return $success; # if query worked this will return 1;
 }
 
+=pod
+
+=head3 is_study_unique($dbh, $update, $Archivemd5)
+
+Verifies that the DICOM study is already registered in the C<tarchive> tables
+using the StudyUID field of the DICOM files. If the study is already present in the
+C<tarchive> tables but C<-clobber> was not when running <dicomTar.pl> or that we are
+using <dicomSummary.pl>, it will return the appropriate error message.
```

Comment:
Does it need the `C` here?  `using C<dicomSummary.pl>, ...`

Reply:
The `C<>` is for the perldoc formatting so that the text is formatted in code and it converts it automatically to `tarchive` when creating the markdown file automatically using `pod2markdown`. This `C<>` convention is used all throughout the perl documentation.

Hope this helps.